node/popn.go

```diff
@@ -1239,6 +1285,36 @@ func (nd *node) Add(ctx context.Context, dag ipldformat.DAGService, buf io.Reade
 	return n.Cid(), nil
 }
 
+// selectDAGParams returns the best chunk params according to the file's type
+func selectDAGParams(filepath string, buf io.ReadSeeker) (
+	chunker chunk.Splitter,
+	layout func(*helpers.DagBuilderHelper) (ipldformat.Node, error),
+) {
+	fileType := utils.DetectFileType(filepath, buf)
+
+	switch fileType {
+	case utils.Audio, utils.Video:
+		chunkSize := int64(1_000_000)
```

Comment:
I'm debating using MiB instead of MB here because Filecoin storage is usually priced in GiB although it might seem a bit less intuitive to some users